README.md

```diff
@@ -20,4 +37,32 @@ any values in the project's config.py.
 4. To run the development server, execute the following:
 ```bash
 % env/bin/python run.py
-```
\ No newline at end of file
+```
+
+## Static assets development
+
+Javascript and SASS assets are built against Node.js v8.9.3. Usage of
+[nvm](https://github.com/creationix/nvm) is a convenient way to use a specific
+version:
+
+```bash
+nvm use v8.9.3
+```
+
+Node.js dependencies are installed via:
+
+```bash
+npm install
+```
+
+And a development server hosting both the Python and static assets run with:
+
+```bash
+npm run watch
+```
+
+To build the complete distributable package, built to `./dist`:
+
+```bash
+npm run build
```

Comment:
Is this a step that would typically be done during the build process on Jenkins? Can you explain how to run locally with built assets? This can be helpful to make sure the entire build process works and to debug issues that only occur with built assets.

Reply:
Yes, that would be the production build. If we wanted to host those locally, there would be an extra step to run the server. I'll add docs for that.